tests/FakeItEasy.Tests/Core/CallOrderingContextTests.cs

```diff
@@ -0,0 +1,146 @@
+﻿namespace FakeItEasy.Tests.Core

```

Comment:
I struggle with these sorts of things as well. I tend to think that things the clients will need to refer to should be in the root namespace, and other things shouldn't. It's probably not so simple. Especially since we have the `internal` keyword to hide things from the clients anyhow. Still, I think I'd be inclined to go with Core.